test/e2e/e2e_suite_test.go

```diff
@@ -102,6 +105,11 @@ func TestE2E(t *testing.T) {
 var _ = SynchronizedBeforeSuite(func() []byte {
 	// Before all ParallelNodes.
 
+	if (os.Getenv("PAUSE_FOR_DEBUGGER_ATTACH") == "true") {
```

Comment:
is this documented anywhere? Any instructions for debugging?

Reply:
After our 1:1, added generic debugging content to e2e README.md on this.